ServerCore/DataModel/Feedback.cs

```diff
@@ -11,7 +11,7 @@ public class Feedback
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int ID { get; set; }
         public Puzzle Puzzle { get; set; }
-        public User Submitter {get; set;}
+        public PuzzleUser Submitter {get; set;}
```

Comment:
Do we need a DB migration for this?

Reply:
I'm not entirely sure. Adding Morgan and Megan because hopefully one of them knows and can tell me what to do if the answer is yes.

Reply:
@morganbr ? @asyasky ?

Reply:
It looks like User->PuzzleUser was committed by Megan a while ago. I must not have been on a current version. So this should just merge cleanly without a data migration.